src/components/button/Button.tsx

```diff
@@ -38,8 +44,8 @@ const DefaultModeStyles: {[key in ButtonMode]: ButtonSettings} = {
 };
 
 type ButtonTypeAwareProps =
-  | {text?: string; type: 'inline'}
   | {text: string; type?: 'block'}
+  | {text?: string; type: 'inline'}
   | {text: string; type: 'pill'};
```

Comment:
In Figma now the types are `large | medium | small`, and not `block | inline | pill`.